### PR TITLE
XRSession: Use single page for events

### PR DESCRIPTION
**Pull request by Elchi3**
#### Summary

For all XRSession events, use a single "event type" page. Redirect all onXXX event handler pages to these.

#### Motivation

One place to learn about XRSession events instead of two for each event.

#### Related issues
 The BCD work is done in https://github.com/mdn/browser-compat-data/pull/13597

#### Metadata

- [ ] Adds a new document
- [x] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/api/xrsession/visibilitychange_event/index.md [hunk 1]
```diff
@@ -29,31 +29,32 @@ browser-compat: api.XRSession.visibilitychange_event
...
+
+### Trigger
+
+Triggered when an {{domxref("XRSession")}} becomes visible or hidden, or when it becomes visible but not currently focused.
 
 When the `XRSession` receives this event, the visibility state has already been changed.
 
+### Event type
+
+An {{domxref("XRSessionEvent")}} which, in addition to the generic {{domxref("Event")}} class, includes the following properties:
```
Comment (jpmedley) [inline files/en-us/web/api/xrsession/visibilitychange_event/index.md:52]: Where are the properties?

**Review by jpmedley — CHANGES_REQUESTED** [review]

File: files/en-us/web/api/xrsession/visibilitychange_event/index.md (hunk 1 above)
Comment (Elchi3) [reply files/en-us/web/api/xrsession/visibilitychange_event/index.md]: Forgot them here, thanks, added!

File: files/en-us/web/api/xrsession/selectstart_event/index.md [hunk 2]
```diff
@@ -24,32 +24,40 @@ browser-compat: api.XRSession.selectstart_event
...
-
-For details on how the {{domxref("XRSession.selectstart_event", "selectstart")}}, {{domxref("XRSession.select_event", "select")}}, and {{domxref("XRSession.selectend_event", "selectend")}} events work, and how you should react to them, see {{SectionOnPage("/en-US/docs/Web/API/WebXR_Device_API/Inputs", "Primary actions")}}.
+The [WebXR](/en-US/docs/Web/API/WebXR_Device_API) event **`selectstart`** is sent to an {{domxref("XRSession")}} when the user begins a [primary action](/en-US/docs/Web/API/WebXR_Device_API/Inputs#primary_action) on one of its input sources.
+
+## Syntax
+
+Use the event name in methods like {{domxref("EventTarget.addEventListener", "addEventListener()")}}, or set an event handler property.
+
+```js
+addEventListener('selectstart', () => { });
```
[the remark this answers is not in the document]
Comment (Elchi3) [reply files/en-us/web/api/xrsession/selectstart_event/index.md]: Done everywhere, thank you!

File: files/en-us/web/api/xrsession/inputsourceschange_event/index.md [hunk 3]
```diff
@@ -26,37 +26,58 @@ browser-compat: api.XRSession.inputsourceschange_event
...
+oninputsourceschange = event => { });
+```
+
+## Description
+
+### Trigger
+
+Triggered when the set of available WebXR input devices changes.
+
+### Event type
```
[the remark this answers is not in the document]
Comment (Elchi3) [reply files/en-us/web/api/xrsession/inputsourceschange_event/index.md]: I've now added both 'Event type' and 'Event properties'. The structure for event page is currently under discussion, so there is no rule to follow yet - I'm just making things up. See https://github.com/mdn/content/discussions/10544

**Review by jpmedley — APPROVED** [review]

**Pull request merged**